Automattic/wp-calypso
# Feature-example: Fix feature example gradient going over masterbar

The gradient we use over the components that are being displayed on `feature-example` had a z-index that was bigger than the one masterbar has, so it was bleeding over it when the viewport size were small enough:
Before
![image](https://cloud.githubusercontent.com/assets/1554855/12390639/1d4eeebe-bde2-11e5-9e1a-eef83dff4491.png)

After:
![image](https://cloud.githubusercontent.com/assets/1554855/12390648/3a822db6-bde2-11e5-9a78-9dc47f88c296.png)


How to test
=====
1. You need a jetpack site with disabled manage.
2. Selected that site, and go to http://calypso.dev:3000/design/jetpacktests.tk::wp2
3. Resize your browser so the grandient over the themes is bigger than the viewport. It should go under masterbar.

/cc @mtias
